lib/ui/product/widget/product_description.dart

```diff
@@ -0,0 +1,336 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flipper/utils/HexColor.dart';
+
+class ProductDescription extends StatefulWidget {
```

Comment:
We decided to not use Stateful Widget unless there is a convincing reason but I do not see any here